homeassistant/components/tado/water_heater.py

```diff
@@ -98,6 +100,9 @@ def create_water_heater_entity(tado, name: str, zone_id: int, zone: str):
 
     supports_temperature_control = capabilities["canSetTemperature"]
 
+    min_temp: float | None
+    max_temp: float | None
```

Comment:
`min_temp` and `max_temp` are property methods, not attributes.
While this fixes the MyPy warning, it suppresses the real issue at hand.